src/main/scripts/rpki-publication-server.sh

```diff
@@ -29,8 +29,8 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #
 
-EXECUTION_DIR=`dirname "$BASH_SOURCE"`
-cd ${EXECUTION_DIR}
+EXECUTION_DIR=$(dirname "${BASH_SOURCE[0]}")
+cd "${EXECUTION_DIR}" || exit 1
```

Comment:
Instead you could `set -e` on top of the script.

Reply:
Not sure if that interacts well with `pkill && rm` etc, or if those lines need to be modified as well do deal with failures. So for now I'll leave it as is.